curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,494 @@ Which of the following is the default value of the `position` property?
 
 `static`
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which position value positions the element according to the normal flow, then offsets it without affecting other elements?
+
+#### --distractors--
+
+`sticky`
+
+---
+
+`absolute`
+
+---
+
+`static`
+
+#### --answer--
+
+`relative`
+
+### --question--
+
+#### --text--
+
+Which position value positions the element relative to the nearest positioned ancestor?
+
+#### --distractors--
+
+`static`
+
+---
+
+`relative`
+
+---
+
+`sticky`
+
+#### --answer--
+
+`absolute`
+
+### --question--
+
+#### --text--
+
+When you apply top: 50% to an absolutely positioned element, what does that 50% refer to?
+
+#### --distractors--
+
+`50% of the element’s own height`
+
+---
+
+`50% of the viewport height`
+
+---
+
+`50% of the nearest ancestor’s width`
+
+#### --answer--
+
+`50% of the nearest positioned ancestor’s height`
+
+### --question--
+
+#### --text--
+
+If you set position: absolute on an inline element (like a <span>), what does its display type become according to the spec?
+
+#### --distractors--
+
+`It remains inline by default`
+
+---
+
+`It becomes display: table`
+
+---
+
+`It becomes a flex container`
+
+#### --answer--
+
+`It’s computed as a block‐level box`
```

Comment:
these don't need to be wrapped in backticks. please fix. 
plus they need periods.